src/main/java/io/candydoc/plugin/save_ubiquitous_language/file/Manifest.java

```diff
@@ -0,0 +1,28 @@
+package io.candydoc.plugin.save_ubiquitous_language.file;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Manifest {
+  public static final Path DESTINATION_FOLDER = Path.of("target/candy-doc");
+  private final List<Path> jsonPaths;
+  private final ObjectMapper serializer;
+
+  public Manifest(ObjectMapper serializer) {
+    jsonPaths = new ArrayList<>();
+    this.serializer = serializer;
+  }
+
+  public void addFile(Path resolve) {
+    jsonPaths.add(resolve);
+  }
+
+  public void save(String extension) throws IOException {
```

Comment:
save should not have extension. We already have paths and an object mapper

Reply:
I corrected it.